### PR TITLE
Use TimeSpan's static method to get total milliseconds

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Logic/TimeCode.cs [hunk 1]
```diff
@@ -95,8 +93,7 @@ public int Minutes
             }
             set
             {
-                var ts = TimeSpan;
-                TimeSpan = new TimeSpan(0, ts.Hours, value, ts.Seconds, ts.Milliseconds);
+                _totalMilliseconds = TimeSpan.FromMinutes(value).TotalMilliseconds;
```
Comment (niksedk) [inline src/Logic/TimeCode.cs]: I think this is wrong... it's not total Minutes...

Comment (ivandrofly) [inline src/Logic/TimeCode.cs]: > I think this is wrong... it's not total Minutes...

Total minutes :confused:

Comment (niksedk) [inline src/Logic/TimeCode.cs]: I think the above code will remove hours, seconds and milliseconds... right?
(I've also added some unit tests regarding this)

Comment (ivandrofly) [inline src/Logic/TimeCode.cs]: I see, I thought by re-assigning we should 0 out every existing time.
:+1:

Comment (niksedk) [inline src/Logic/TimeCode.cs]: only for "TotalMinutes" (if that was there) - but it's covered by unit tests now, so that's nice :)

**Pull request closed without merging**